File: FoodSuggestions/app/src/main/java/com/example/foodsuggestions/models/Recipe.java
package com.example.foodsuggestions.models;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class Recipe {
    @SerializedName("title")
    public String title;

    @SerializedName("image")
    public String image;

    @SerializedName("summary")
    public String summary;

    @SerializedName("extendedIngredients")
    public ArrayList<Ingredients> extendedIngredients;

    @SerializedName("analyzedInstructions")
    public ArrayList<Instructions> analyzedInstructions;


   // Ingredients extendedIngredients;

}

File: FoodSuggestions/app/src/main/java/com/example/foodsuggestions/models/Recipe.kt
package com.example.foodsuggestions.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize
import java.util.ArrayList

@Parcelize
data class Recipe(
        @SerializedName("title") val title: String,
        @SerializedName("image") val image: String,
        @SerializedName("summary") val summary: String,
        @SerializedName("sourceUrl") val sourceUrl: String,
        @SerializedName("dairyFree") val dairyFree: String,
        @SerializedName("glutenFree") val glutenFree: String,
        @SerializedName("dishTypes") val dishTypes : ArrayList<String>,
        @SerializedName("extendedIngredients") val extendedIngredients: ArrayList<Ingredients>,
        @SerializedName("analyzedInstructions") val analyzedInstructions: ArrayList<Instructions>
): Parcelable

